### PR TITLE
initial pass for 4.6 enablement

### PR DESCRIPTION
**Pull request by rphillips**

### CODE REVIEW

**Comment by rphillips** [comment]
@tkashem ptal

File: manifests/4.6/clusterresourceoverride-operator.v4.6.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+    createdAt: 2019/11/15
+    description: An operator to manage the OpenShift ClusterResourceOverride Mutating Admission Webhook Server
+    healthIndex: B
+    repository: https://github.com/openshift/cluster-resource-override-operator
+    support: Red Hat
+  name: clusterresourceoverride-operator.v4.6.0
+  namespace: clusterresourceoverride-operator
+spec:
+  relatedImages:
+  - name: operator.v4.5
```
Comment (tkashem) [inline manifests/4.6/clusterresourceoverride-operator.v4.6.0.clusterserviceversion.yaml:20]: should this be `operator.v4.6`?

File: manifests/4.6/clusterresourceoverride-operator.v4.6.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+    image: quay.io/openshift/clusterresourceoverride-rhel7-operator:4.6
+  - name: operand.v4.5
```
Comment (tkashem) [inline manifests/4.6/clusterresourceoverride-operator.v4.6.0.clusterserviceversion.yaml:22]: `operand.v4.6`

Comment (rphillips) [reply manifests/4.6/clusterresourceoverride-operator.v4.6.0.clusterserviceversion.yaml]: thanks... updated.

File: manifests/clusterresourceoverride-operator.package.yaml [hunk 3]
```diff
@@ -4,3 +4,5 @@ channels:
     currentCSV: clusterresourceoverride-operator.v4.4.0
   - name: "4.5"
     currentCSV: clusterresourceoverride-operator.v4.5.0
+  - name: "4.6"
```
Comment (tkashem) [inline manifests/clusterresourceoverride-operator.package.yaml:7]: @rphillips try removing both 4.4 and 4.5 channels from the package, this should set 4.6 as the default channel. 

If there is only one channel in a package then it is considered as the default channel - https://github.com/operator-framework/operator-registry/blob/bf0c8f1590d21e54e97c1446b09fae72ade51280/pkg/registry/types.go#L82

**Comment by rphillips** [comment]
e2e-aws-olm went green on the last run... this PR should be ready for review

**Comment by rphillips** [comment]
For reference, https://github.com/operator-framework/operator-registry/pull/332 added a copy to the database. The image's WORKDIR was pointing to / not allowing the container to write to the root of the container.

**Comment by sjenning** [comment]
4.6 ART build is busted so... anything is an improvement!

/lgtm

**Pull request merged**